examples/tcprtt/tcprtt.c

```diff
@@ -0,0 +1,85 @@
+// +build ignore
+
+#include "common.h"
+
+#include "bpf_endian.h"
+#include "bpf_helpers.h"
+#include "bpf_tracing.h"
+
+#define AF_INET 2
+
+char LICENSE[] SEC("license") = "Dual MIT/GPL";
+
+struct sock_common {
```

Comment:
Could you document what the struct defs are about? I'm guessing you made a simplified copy of the kernel structure, but it would be nice to have that spelled out.